google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -453,6 +468,22 @@ def autoclass_enabled= toggle
           patch_gapi! :autoclass
         end
 
+        ##
+        # Update method to update all attributes of autoclass of a bucket
+        # It accepts params as a Hash of attributes in the following format:
+        #
+        #     { enabled: true, terminal_storage_class: "ARCHIVE" }
+        #
```

Comment:
The generic method is sufficient, thanks for the update. Just one nit: consider adding a comment on the newly added field
`terminal_storage_class field is optional and defaults to NEARLINE if not otherwise specified. Valid terminal_storage_class values are NEARLINE and ARCHIVE`